### ISSUE TITLE
New VAMC Facility: Chicago VA Mobile Clinic (BOLO) ON HOLD

### ISSUE THREAD

**Issue by ian-sears** [department-of-veterans-affairs/va.gov-cms#18426]
## Intake
- [ ] What triggered this runbook?
Trigger: Flag in CMS

- [ ] Link to associated help desk ticket (if applicable)
Help desk ticket: [VAHELP-7422](https://va-gov.atlassian.net/browse/VAHELP-7422)

- [ ] Name of submitter (if applicable)
Submitter: Karen.Sullivan6@va.gov

- [ ] If the submitter is an editor, send them links to any relevate KB articles for the VAMC Facility product.
KB articles: [How do I add a facility to my health care system?](https://prod.cms.va.gov/help/vamc/about-locations-content-for-vamcs/how-do-i-add-a-facility-to-my-health-care-system)
[How to edit a VAMC Facility](https://prod.cms.va.gov/help/va-medical-centers-vamcs/about-locations-content-for-vamcs/how-to-edit-a-vamc-facility)

- [ ] Link to facility in production:
Facility CMS link: [https://prod.cms.va.gov/chicago-health-care/locations/chicago-va-mobile-clinic](https://prod.cms.va.gov/chicago-health-care/locations/chicago-va-mobile-clinic)
Facility API ID: vha_537QB

### New VAMC Facility
Please refer to the Knowledge Base article titled "How do I add a facility to my health care system?" for more information: https://prod.cms.va.gov/help/vamc/about-locations-content-for-vamcs/how-do-i-add-a-facility-to-my-health-care-system

#### CMS help desk steps
**Note: If the help desk is waiting on information from the facility staff or editor, add the "Awaiting editor" flag to the facility with a log message that includes a link to this ticket. Remove the flag when the ticket is ready to be worked by the Facilities team. Be sure to preserve the current moderation state of the node when adding or removing the flag.**
- [x] Become aware that the new facility is now on the Facility API (typically, via a Flag, but this may come in as a helpdesk ticket).
- [x] If the editor has followed the steps from the above Knowledge Base article and included which section and VAMC the facility belongs to (i.e. VA Pittsburgh), great!
  - If not, please check with the editor or VHA digital media regarding what section and VAMC it belongs to.
    - [x] Update the Section (default is "VAMC facilities", but it should be a VAMC system in a VISN) and VAMC system field accordingly.
- [x] Communicate with editor (cc VHA Digital Media) to give them go-ahead to complete the content, with this [KB article](https://prod.cms.va.gov/help/vamc/about-locations-content-for-vamcs/how-do-i-add-a-facility-to-my-health-care-system).

- [ ] When editor has prepared content and let help desk know, proceed to the remaining steps.

#### Drupal Admin steps (CMS Engineer or Help desk) _Help desk will complete these steps or escalate to request help from CMS engineering._
- [x] Update the facility **Meta description** field, using the following format: "Get address and hours, parking and transportation information, and health services offered at [facility name]."
- [ ] Move the facility link in the health care system menu to its place in the alphabetized list (medical centers first, then clinics).
- [ ] Drupal Admin bulk publishes nodes and facility.
- [ ] Drupal Admin edit facility node and remove `New facility` flag and save node.
- [ ] Contact Lighthouse via Slack at #cms-lighthouse channel that this facility requires a canonical link in the following format (replacing the placeholder data with the actual API Id and VA.gov URL):
  - `vha_537QB,https://www.va.gov/chicago-health-care/locations/chicago-va-mobile-clinic/`
- [ ] Add the "Awaiting CSV" flag to the facility node with a revision log message that includes a link to this ticket.
- [ ] Let Help desk know this has been done, if not done by Help desk.

#### Wait (days or weeks, potentially)
- [ ] After the canonical link has been added to the websites.csv and you have confirmation from Lighthouse that the CSV has been deployed, validate that the change has deployed by checking that the Facility Locator has been updated with the new url.
- [ ] Update this ticket with a comment that the CSV change has been deployed.
- [ ] Remove the "Awaiting CSV" flag to the facility node with a revision log message that includes a link to this ticket.

#### CMS Help desk (wrap up)
- [ ] Notify editor and any other stakeholders.

[VAHELP-7422]: https://va-gov.atlassian.net/browse/VAHELP-7422?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by omahane** [comment]
Closing, per new guidance on VA Mobile Clinics.
